### ISSUE TITLE
Help find a home for this lost Washington.md data

### ISSUE THREAD

**Issue by ubershmekel** [2020PB/police-brutality#749]
This PR https://github.com/2020PB/police-brutality/pull/745 required deleting this data. I don't know who was using it or what for. We can throw it away as far as I'm concered. But if it's useful, then it should be rolled into the specific case descriptions.

```
Additional Seattle OPA Case Numbers | 

[Source - Twitter @SeattleOPA](https://twitter.com/SeattleOPA)


* 2020OPA-0323 - May 29 - "Punching a person on the ground who was being arrested"
* 2020OPA-0324 - May 30 - "Placing the knee on the neck area of two people who had been arrested"
* 2020OPA-0326 - May 30 - "Pepper spraying peaceful protesters"
* 2020OPA-0328 - May 30 - "The use of flashbangs, including causing a significant thumb injury"
* 2020OPA-0332 - June 1 - "A confrontation between an officer and a male"
* 2020OPA-0335 - June 1 - "Homeless man hit by an SPD flashbang while sleeping"
```

**Comment by ContributingThrowaway** [comment]
I think having OPA numbers would be valuable -- the investigations are being concluded with unusual speed (https://www.seattle.gov/opa/case-data/demonstration-complaint-dashboard; note that all the investigations listed above are on the dashboard) and it would be nice to be able to go back later and see which of these cases of misconduct result in actual officer discipline. But given that these descriptions often will not suffice to uniquely identify the incident in question, suggest that associating OPA number (if it ever gets done) will have to wait until reports are made public (which will happen once investigations are complete; that should be within 90 days of the complaint but could be much much sooner).

**Comment by emarcey** [comment]
I agree with @ContributingThrowaway , there are probably a dozen incidents that match "Pepper spraying peaceful protestors" and "A confrontation between an officer and a male"

But once those reports come out, then maybe we could return. I thought we had an incident for the homeless man, but it must've been a homeless man gassed in some other city 🤦‍♀️
